jasper.proto

```diff
@@ -190,6 +190,12 @@ message DownloadInfo {
     ArchiveOptions archive_opts = 3;
 }
 
+message WriteFileInfo {
+    string path = 1;
+    bytes data = 2;
```

Comment:
shouldn't this be a streaming API?

Reply:
I changed the client/server to use client streaming.